one_fm/www/employee-info/index.html

```diff
@@ -30,7 +30,7 @@ <h4 class="font-weight-bold mb-2">Employee Details</h4>
             <div class="card-body">
               <div class="d-flex flex-column align-items-center text-center">
                 <img
-                  src="{{ employee.image }}"
+                  src="{{ employee.image if employee.image else 'https://www.huntregional.org/wp-content/uploads/2023/08/profile-photo-placeholder.png' }}"
```

Comment:
@ibtesam-fm Add this placeholder in public directory and use that instead of a third-party link. Rest looks good.

Reply:
Sure @ks093